eng/pipelines/performance/templates/build-perf-sample-apps.yml

```diff
@@ -1,35 +1,21 @@
 parameters:
   osGroup: ''
-  osSubgroup: ''
-  archType: ''
-  buildConfig: ''
-  runtimeFlavor: ''
-  helixQueues: ''
-  targetRid: ''
+  flavor: 'mono' # Currently only used for Android Hello World app
   nameSuffix: ''
-  platform: ''
-  shouldContinueOnError: ''
-  rootFolder: ''
-  includeRootFolder: ''
-  displayName: ''
-  artifactName: ''
-  archiveExtension: ''
-  archiveType: ''
-  tarCompression: ''
   hybridGlobalization: False
 
 steps:
 # Build Android sample app
   - ${{ if eq(parameters.osGroup, 'android') }}:
-    - script: make run TARGET_ARCH=arm64 DEPLOY_AND_RUN=false
+    - script: make run TARGET_ARCH=arm64 DEPLOY_AND_RUN=false RUNTIME_FLAVOR=${{ parameters.flavor }}
```

Comment:
Do we want to have this parametrised? 

I imagined we would have separate steps for Mono and for CoreCLR for the fact that different runtimes will have different configurations, e.g., CoreCLR does not have interpreter, Mono does, CoreCLR has R2R, Mono doesn't, etc.

I suggest splitting it up into separate steps.

Reply:
I believe that the runtime build doesn't change, we just set the makefile variable to enable the configuration. The issue might be that there are separate variables for CoreCLR and Mono as Ivan mentioned...

CoreCLR has R2R, Mono has AOT, Mono has interpreter (CoreCLR not).

Reply:
That makes sense, I will separate them out between mono and coreclr to help make the additional options for each more clear in the future.